include/fiction/algorithms/physical_design/apply_gate_library.hpp

```diff
@@ -122,15 +123,23 @@ class apply_gate_library_impl
      *
      * @tparam Params Type of the Parameters used for the SiDB on-the-fly gate library.
      * @param params Parameters used for the SiDB on-the-fly gate library.
+     * @param lyt Optional cell-level layout to add defects to.
```

Comment:
This is not mentioned in the docstring at all and feels kinda out of place. This function has nothing to do with defects. It applies a gate library. Seems confusing.

Reply:
@marcelwa, I know exactly what you mean. But I do not know how to make it better. Do you have any ideas?

Reply:
We could add another function that by name reflects its applicability to defective surfaces